src/main/java/be/seeseemelk/mockbukkit/ServerMock.java

```diff
@@ -111,6 +112,7 @@
 {
 	private static final String BUKKIT_VERSION = "1.16.5";
 	private static final String JOIN_MESSAGE = "%s has joined the server.";
+	private static final String QUIT_MESSAGE = "%s has exit the server.";
```

Comment:
```suggestion
	private static final String QUIT_MESSAGE = "%s has exited the server.";
```
Small typo I noticed while looking through this.